ranch/transit.py

```diff
@@ -594,17 +626,26 @@ def snap_stop_to_node(self):
         RanchLogger.info('Snapping gtfs stops to roadway node...')
 
         # get rid of motorway nodes
-        non_motorway_links_df = self.roadway_network.links_df[
-            ~self.roadway_network.links_df.roadway.isin(["motorway", "motorway_link"])
+        motorway_links_df = self.roadway_network.links_df[
+            self.roadway_network.links_df.roadway.isin(["motorway", "motorway_link", 'trunk', 'trunk_link'])
         ].copy()
+        motorway_nodes_list = motorway_links_df.fromIntersectionId.tolist() + motorway_links_df.toIntersectionId.tolist()
+
+        ag100plus_links_df = self.roadway_network.links_df[
+            self.roadway_network.links_df.assign_group>=100
+        ].copy()
+        ag100plus_nodes_list = ag100plus_links_df.fromIntersectionId.tolist() + ag100plus_links_df.toIntersectionId.tolist()
```

Comment:
You are excluding any nodes that touch the ag100+ links, which is probably an overkill, because a lot of these nodes could be on a primary street.